snippets/eruby.snippets

```diff
@@ -46,8 +46,9 @@ snippet conf
 	<% content_for :${1:head} do %>
 		${0}
 	<% end %>
-snippet cs
-	<%= collection_select <+object+>, <+method+>, <+collection+>, <+value_method+>, <+text_method+><+, <+[options]+>, <+[html_options]+>+> %>
```

Comment:
I guess the <+ and +> is outdated, also it's not this helpful to use this snippet.